backend/models/BlacklistToken.js

```diff
@@ -10,52 +10,52 @@ const __filename = fileURLToPath(import.meta.url);
 
 const blacklistCollection = getBlacklistCollection;
 
-const BlacklistToken = {
-  addTokenToList: async (token, expires) => {
-    try {
-      await blacklistCollection.insertOne({
-        token,
-        expires: Date.now(expires * 1000),
-      });
-    } catch (error) {
-      throw await errorHelpers.transformDatabaseError(
-        error,
-        __filename,
-        'addTokenToList',
-      );
-    }
-  },
+const blackListToken = {
+    addTokenToList: async (token, expires) => {
+        try {
+            await blacklistCollection.insertOne({
+                token,
+                expires: Date.now(expires * 1000),
+            });
+        } catch (error) {
+            throw await errorHelpers.transformDatabaseError(
+                error,
+                __filename,
+                'addTokenToList',
+            );
+        }
+    },
 
-  getToken: async (token) => {
-    try {
-      return await blacklistCollection.findOne(
-        { token },
-        { projection: { token: 1 } },
-      );
-    } catch (error) {
-      throw await errorHelpers.transformDatabaseError(
-        error,
-        __filename,
-        'findOne',
-      );
-    }
-  },
+    getToken: async (token) => {
+        try {
+            return await blacklistCollection.findOne(
+                { token },
+                { projection: { token: 1 } },
+            );
+        } catch (error) {
+            throw await errorHelpers.transformDatabaseError(
+                error,
```

Comment:
spacing fix here too

Reply:
I made the updates requested, I have actually implemented refresh and access tokens to be used this week. The plan is to remove the block list.

Once I have verification via email setup, I plan to make another PR which will include the new refreshToken functionality including testing for the model. I can make the PR with the included refresh token functionality before I implement the email verification  if you would like.

You  can let me know on discord, I am TimE in the litter.ai room

Thanks!

Reply:
Wonderful. Ok sounds great. We are getting organized way more on ops side and might have more options there for volunteers to help as well I will keep you updated. Email setup might take some time, I know David has a domain ready to go and we should have more bandwidth to get these streamlined soon.